djangochannelsrestframework/observer/model_observer.py

```diff
@@ -133,37 +136,31 @@ def post_change_receiver(self, instance: Model, action: Action, **kwargs):
 
         self.get_observer_state(instance).current_groups = new_group_names
```

Comment:
since this method is called before the transition commits I am not sure we should be updating the `current_groups`? maybe these group names shoudl only be updated in the on commit?

Also what is the expected behavior here if we have multiple changes to the same instance within a transaction. Does this result in a single message being sent or does it result in multiple Update messages being sent?

Reply:
If we update `current_groups` in the on commit and then we ensure only the latest set of messages for the connection is set.  Maybe we store messages on self mapping a connection ID to the message.